presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -0,0 +1,101 @@
+"""
+AnalyzerResult is the exact copy of the recognizer result.
+
+Represents the findings of detected entity.
+"""
+from presidio_anonymizer.entities.invalid_exception import InvalidParamException
+
+
+class AnalyzerResult(object):
+    """
+    AnalyzerResult is a duplication of the recognizer result.
+
+    Suppose to validate and compare an recognizer result object.
```

Comment:
I would suggest to leave out the word `Suppose`

Reply:
Done